### PR TITLE
Minor additions

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas/geometry/brep/brep.py [hunk 1]
```diff
@@ -489,6 +487,26 @@ def from_pipe(cls, curve, radius, thickness=None):
         """
         return from_pipe(curve, radius, thickness=thickness)
 
+    @classmethod
+    def from_plane(cls, plane, domain_u=(-1, +1), domain_v=(-1, +1)):
+        """Construct a Brep from a set of planes.
```
Comment (gonzalocasas) [inline src/compas/geometry/brep/brep.py]: From one plane of from a set of planes? If the latter, the method should be called `from_planes`

File: src/compas/geometry/brep/__init__.py [hunk 2]
```diff
@@ -72,6 +72,11 @@ def from_pipe(*args, **kwargs):
     raise PluginNotInstalledError
 
 
+@pluggable(category="factories")
+def from_plane(*args, **kwargs):
+    raise PluginNotInstalledError
+
```
Comment (gonzalocasas) [inline src/compas/geometry/brep/__init__.py:78]: Is there a rhino implementation that we could add right away as well? Or will this be only OCC?

**Review by gonzalocasas — APPROVED** [review]
lgtm + a couple of questions

File: src/compas/geometry/brep/__init__.py (hunk 2 above)
Comment (tomvanmele) [reply src/compas/geometry/brep/__init__.py:78]: dunno. will check

Comment (tomvanmele) [reply src/compas/geometry/brep/__init__.py:78]: no, Rhino doesn't have a `CreateFromPlane` or similar...

**Pull request merged**